src/frontend/src/pages/GanttPage/GanttChartComponents/GanttChartFilters.tsx

```diff
@@ -116,10 +123,10 @@ const GanttChartFilters = ({
         maxWidth: { xs: '100%', md: '30rem' }
       }}
     >
-      <FilterRow label="Cars" buttons={carHandlers} />
-      <FilterRow label="Team Category" buttons={teamCategoriesHandlers} />
-      <FilterRow label="Team" buttons={teamsHandlers} />
-      <FilterRow label="Overdue" buttons={[{ filterLabel: 'Overdue', handler: overdueHandler }]} />
+      <FilterRow label="Car" buttons={carHandlers} />
+      <FilterRow label="Subteam" buttons={teamTypeHandlers} />
+      <FilterRow label="Team" buttons={teamHandlers} />
+      <FilterRow label="Overdue" buttons={[{ filterLabel: 'Overdue', handler: overdueHandler, defaultChecked: false }]} />
       <FilterButtons />
     </Grid>
   );
```

Comment:
The reset button is resetting the filter, but not deactivating the button:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/ed2bf050-4210-4834-a303-a3d2a3cbc5dc)